x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_logistics/step_logistics.tsx

```diff
@@ -501,7 +583,7 @@ export const RestoreSnapshotStepLogistics: React.FunctionComponent<StepProps> =
           <FormattedMessage
             id="xpack.snapshotRestore.restoreForm.stepLogistics.includeGlobalStateDescription"
             defaultMessage="Restores templates that don’t currently exist in the cluster and overrides
-              templates with the same name. Also restores persistent settings."
+              templates with the same name. Also restores persistent settings and data streams."
```

Comment:
I don't think this or the current message is correct.

Restoring the global cluster state restores any resources stored in the cluster state, including:

- Persistent cluster settings
- Index templates
- Ingest pipelines
- ILM lifecycle policies

Restoring the global cluster state overwrites any existing cluster settings, templates, pipelines, and lifecycle policies whose names match those in the snapshot.

It does _not_ restore data streams.

Reply:
Right, this should have been removed! Thanks for flagging :+1: